.github/workflows/on_release.yml

```
@@ -4,10 +4,19 @@
+  # Specifies API environment for KMP build.
+  # One of [dev|prod] as per configuration of Buildkonfig plugin in :shared:network:* Gradle module.
+  KMP_FLAVOR: prod
+
 jobs:
   release_ios:
     name: iOS Release
     uses: futuredapp/.github/.github/workflows/ios-kmp-selfhosted-release.yml@main
+    with:
+      kmp_swift_package_integration: true
+      kmp_swift_package_path: iosApp/shared/KMP
+      kmp_swift_package_flavor: ${{ env.KMP_FLAVOR }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)